### PR TITLE
Angle gear ratio

### PR DESCRIPTION
**Pull request by Legomaster96734**
fixed the drive going the wrong direction, the wheels all point in the correct direction

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -67,16 +67,16 @@ public static final class Swerve {
     public static final double angleConversionFactor = 360.0 / angleGearRatio;
 
     /* Swerve Profiling Values */
-    public static final double maxSpeed = 4.5; // meters per second
+    public static final double maxSpeed = 3.5; // meters per second orginal value 4.5 FIXME: the speed does not seen to be changing
```
Comment (Legomaster96734) [inline src/main/java/frc/robot/Constants.java:70]: Where did this value come from?

File: src/main/java/frc/robot/Constants.java [hunk 2]
```diff
@@ -26,8 +26,8 @@ public static final class Swerve {
     public static final double openLoopRamp = 0.25;
     public static final double closedLoopRamp = 0.0;
 
-    public static final double driveGearRatio = (6.75 / 1.0); // 6.75:1
-    public static final double angleGearRatio = (12.8 / 1.0); // 12.8:1
+    public static final double driveGearRatio = (6.75 / 1.0);
```
Comment (Apitara12) [inline src/main/java/frc/robot/Constants.java:29]: hello

**Review by leab777 — APPROVED** [review]
good job!

**Review by Apitara12 — APPROVED** [review]
first comment

**Pull request merged**